src/game_inventory.cpp

```diff
@@ -754,6 +754,51 @@ item_location game_menus::inv::holster( player &p, item &holster )
                          hint );
 }
 
+class saw_barrel_inventory_preset: public weapon_inventory_preset
+{
+    public:
+        saw_barrel_inventory_preset( const player &p ) :
+            weapon_inventory_preset( p ) {
+        }
+
+        bool is_shown( const item_location &loc ) const override {
+            return loc->is_gun();
+        }
+
+        std::string get_denial( const item_location &loc ) const override {
+            if( loc->type->gun->barrel_length <= 0 ) {
+                return _( "the barrel is too short" );
```

Comment:
Capitalize first "t" in the "the".

Reply:
The denials are a bit inconsistent for now, but, actually, I was going to use lowercase everywhere.

Reply:
Most of the game uses proper sentences for anything longer than 3 words. Why the exception here?